storage-providers/filecoin-deals/filecoin-programs.md

```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
+
+Here are great existing and working Filecoin dashboards that cover similar topics:
+
+- [Starboard](https://dashboard.starboard.ventures/dashboard)
+- [Filecoin Dune Daily Metrics](https://dune.com/kalen/filecoin-daily-metrics)
+- [Filecoin Pulse (PoC)](https://filecoinpulse.pages.dev/)
+
+## Slingshot
+
+[![GitHub Repo stars](https://img.shields.io/github/stars/filecoin-project/slingshot)](https://github.com/filecoin-project/slingshot)
+![GitHub last commit](https://img.shields.io/github/last-commit/filecoin-project/slingshot)
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+[![Join Slack](https://img.shields.io/badge/join-Slack-purple)](https://filecoinproject.slack.com/archives/C01AZP8BKRQ)
+
+[Slingshot](https://slingshot.filecoin.io) was a program that united Data clients, Data preparers and storage providers in a community to onboard data and share replicas of publicly valuable [_Open Datasets_](https://datasets.filecoin.io). Rather than providing a web interface like Estuary, Slingshot was a program that provides a workflow and tools for onboarding of large open datasets. The Slingshot Deal Engine provided deals to registered and certified storage providers. The data was prepared and uploaded using a tool called Singularity. The program has [clear requirements of a storage provider](https://slingshot.filecoin.io/requirements#participating-as-a-storage-provider-sp) regarding their capabilities.
+
+See the below video for more information on Slingshot:
+
+{% embed url="https://www.youtube.com/watch?v=14sasiEThig" %}
+A overview of the Slingshot project from the Enterprise Storage Provider Accelerator (ESPA).
+{% endembed %}
+
+See also:
+
+- [Slingshot v2](https://v2.slingshot.filecoin.io/)
+- [Slingshot v3](https://v3.slingshot.filecoin.io/)
+- [Slingshot Recovery](https://recovery.slingshot.filecoin.io/)
+- https://evergreen.slingshot.filecoin.io/ ⚠️
+- https://dashboard.slingshot.filecoin.io/ ⚠️
+
+## Filecoin+
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+To enhance the Filecoin's utility, [Filecoin Plus](https://filplus.storage/) introduced a social mechanism that incentivizes participants to maximize useful storage. Data owners used DataCap to make deals with storage providers, who benefited from increased block rewards over time. The Filecoin Plus program involved various stakeholders, including root key holders, notaries, data owners, and storage providers, who collaborated to form a governed community.
+
+## Open Panda
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+[Open Panda](https://openpanda.io/) is a platform for data researchers, analysts, students and enthusiasts to interact with the largest open datasets in the world. Data available through our platform is stored on Filecoin, a decentralized storage network comprised of thousands of independent Storage Providers around the world.
+
+## Data Program
```

Comment:
* I'd actually make the header 

## Dataprograms.org 
and say it was discontinued in April 2024
* Remove the last sentence